web/src/stream_edit_subscribers.js

```diff
@@ -89,6 +89,10 @@ export function enable_subscriber_management({sub, $parent_container}) {
         get_potential_subscribers,
     });
 
+    $pill_container.find(".input").on("input", () => {
+        $parent_container.find(".stream_subscription_request_result").empty();
```

Comment:
Moved this change here from `add_subscribers_pill` as we do not show any message in stream creation form. Maybe we can move it there where we show message in creation form. And we also use different class in group settings.